packages/jest-runner/src/runTest.ts

```diff
@@ -84,6 +84,9 @@ async function runTestInternal(
   resolver: Resolver,
   context?: TestRunnerContext,
 ): Promise<RunTestInternalResult> {
+  // Node's error-message stack size is limited at 10, but it's pretty useful
+  // to see more than that when a test fails.
+  Error.stackTraceLimit = 100;
```

Comment:
when the error happens outside the test sandbox (such as when resolving a dependency in `jest-runtime`)